src/directives/rcdni.js

```diff
@@ -0,0 +1,42 @@
+directives.directive('rcDni',['dniSemanticsValidationService','dniSyntaxValidationService',function(dniSemanticsValidationService,dniSyntaxValidationService){
```

Comment:
What do you think about putting the function in a new line, so you can check the signature in a quick look?